vizier/backend/src/info/vizierdb/commands/plot/ScatterPlot.scala

```diff
@@ -92,6 +92,8 @@ object ScatterPlot extends Command
             xIndex      = series.get[Int](PARAM_X),
             yIndex      = series.get[Int](PARAM_Y),
             filter      = series.getOpt[String](PARAM_FILTER),
+            sort        = false,
+            isBarChart  = false,
```

Comment:
As before, make isBarChart either a default parameter, or move the barChart code to a `Series` method.